server.go

```diff
@@ -260,6 +301,11 @@ func loadState() (*grange.State, int) {
 	return &state, warnings
 }
 
+func cleanupStatsd() {
+	stats.Close()
+	stats = &noopStatsd{}
```

Comment:
why are you resetting stats to the no-op statsd here just to create a new client shortly after?